gravity-forms/gw-cache-buster.php

```diff
@@ -301,6 +301,14 @@ public function ajax_get_form() {
 			return rgar( $_REQUEST, 'form_theme' ) ?: $slug;
 		}, 10, 2 );
 
+		add_filter( 'gpmpn_default_page_' . $form_id, function( $page ) { 
+			if ( rgars( $_REQUEST, 'atts/page' ) && rgget( 'gpmpn_page' ) ) {
+				return rgget( 'gpmpn_page' );
+			}
```

Comment:
Is this block needed? `&gpmpn_page` wasn't getting passed to the AJAX request from what I could see.

It worked for me with the following:

```php
add_filter( 'gpmpn_default_page_' . $form_id, function( $page ) {
    return rgars( $_REQUEST, 'atts/page', $page );
} );
```

Reply:
@claygriffiths  the `&gpmpn_page` is passed [here](https://github.com/gravitywiz/snippet-library/blob/master/gravity-forms/gw-cache-buster.php#L175). And this block is needed to mimic this security feature mentioned  [here](https://gravitywiz.com/documentation/gravity-forms-multi-page-navigation/#custom-starting-page), which I can also see in the code.

Reply:
Roger that!